<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectTreeWidget</class>
 <widget class="QWidget" name="ProjectTreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>31</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="filterEdit">
        <property name="placeholderText">
         <string>Search</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonOptions">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Search Options</string>
        </property>
        <property name="statusTip">
         <string>Search Options</string>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextOnly</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ProjectTreeView" name="projectTreeView">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
  <action name="actionOptionsRegExp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Regular expression pattern</string>
   </property>
  </action>
  <action name="actionOptionsWildcard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wildcard pattern</string>
   </property>
  </action>
  <action name="actionOptionsContainingString">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Containing string</string>
   </property>
   <property name="toolTip">
    <string>Containing string</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProjectTreeView</class>
   <extends>QTreeView</extends>
   <header>ProjectTreeView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>projectTreeView</tabstop>
  <tabstop>filterEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
